exercises/say/test/test_say.c

```diff
@@ -15,7 +15,6 @@ void test_zero(void)
    char *ans = NULL;
    int res = say(0, &ans);
    TEST_ASSERT_EQUAL_INT(0, res);
-   TEST_ASSERT_NOT_NULL(ans);
```

Comment:
Why remove the tests for `NULL`?

Reply:
```TEST_ASSERT_EQUAL_STRING``` checks already for ```NULL``` (see unity.c:1014).

This was first brought to my attention by @patricksjackson in [this PR](https://github.com/exercism/c/pull/274#pullrequestreview-99124342)